theme/p2/taskinfo.p2

```diff
@@ -3,25 +3,31 @@
 {% block content %}
 <ul>
 {% for namespace, jobs in data %}
-<li>{{namespace}}
-  <ul>
+<div class="card">
+  <h2>{{namespace}}</h2>
+  <div class="grid grid-responsive">
     {% for job, groups in jobs %}
-    <li>{{job}}
-      <ul>
-        {% for group, tasks in groups %}
-        <li>{{group}}
-          <ul>
-            {% for task, info in tasks %}
-            <li>{{task}} - {% if info.versions %}Version Information Available{% else %}No Versioning{% endif %}</li>
-            {% endfor %}
-          </ul>
-        </li>
+    <div class="card grid">
+      <h3>{{job}}</h3>
+      {% for group, tasks in groups %}
+      <div class="card grid">
+        <h4>{{group}}</h4>
+        {% for task, info in tasks %}
+        <div class="grid m-3">
+          {{task}} -
```

Comment:
So for each of the various info sources (version is the only one being loaded right now, but there are others that taskinfo knows about) my intent is to display an icon for each: grey if no data is available, green if the status is good, and red if the status is bad.  I think these icons should stack up along the right hand side of each task block, can you make the change to accommodate this?

Reply:
@the-maldridge - Yessir, do you mean similar to the icons on master?

<img width="277" alt="Screen Shot 2021-12-02 at 10 40 27 PM" src="https://user-images.githubusercontent.com/4707505/144546346-bb12028a-504f-4238-86ca-bc09355a5def.png">

Reply:
Similar feel would be nice, no need to try and match them exactly.

Reply:
@the-maldridge - I think I might need some hand holding here just to figure out what iteration over the info sources looks like if thats okay. This data shape feels very bespoke and my go-chops are... formative.

Reply:
You are correct, the shapes are completely bespoke based on what info provider has sourced the information.  The base structure though is a map and so you can use that to decide whether the icon should be grey or not.  Then each one should have a variable that indicates that it is in the "good" state.  For the version checker this is "UpToDate".  For trivy I don't remember and I'll have to check.